strings-csharp/HexToByteArray/Tests/Test.cs

```diff
@@ -0,0 +1,88 @@
+using HexToByteArray;
+
+namespace Tests
+{
+    public class Test
+    {
+        private readonly ConvertHex _convertHex;
```

Comment:
By making the ConvertHex class static, you can remove this line as well as the test constructor

Reply:
Done. Thanks!